evalml/data_checks/invalid_target_data_check.py

```diff
@@ -115,12 +116,12 @@ def validate(self, X, y):
             ...                             'num_null_rows': 2,
             ...                             'pct_null_rows': 50.0},
             ...                 'code': 'TARGET_HAS_NULL'}],
-            ...     'actions': [{'code': 'IMPUTE_COL',
+            ...     'actions': {"action_list": [{'code': 'IMPUTE_COL',
```

Comment:
Mega nit: double quotes?

Reply:
*sobs in wishing there were a doc linter*

Just went through and changed everything to double quotes :))